test-data/unit/python2eval.test

```diff
@@ -396,11 +403,12 @@ def f(x: unicode) -> int: pass
 def f(x: bytearray) -> int: pass
 [out]
 _program.py:2: error: No overload variant of "f" matches argument types [builtins.int]
+_program.py:5: error: No overload variant of "f" matches argument types [builtins.bytes]
 
 [case testByteArrayStrCompatibility_python2]
-def f(x): # type: (str) -> None
+def f(x): # type: (bytes) -> None
```

Comment:
Note that bytearray() is compatible with bytes, but not with str, hence the change here.